flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/DataFormatUtil.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.dataformat;
+
+import org.apache.flink.api.common.typeutils.base.ComparatorUtil;
+import org.apache.flink.api.common.typeutils.base.DateComparator;
+import org.apache.flink.core.memory.MemorySegment;
+
+import java.sql.Date;
+import java.sql.Time;
+import java.sql.Timestamp;
+
+/**
+ * Util for data formats.
+ */
+public class DataFormatUtil {
```

Comment:
Can we separate this class by functionality? One big `DataFormatUtil` seems too large and make developer difficult to tell what exactly this utility provides.

Reply:
Divide it into NormalizedKeyUtil, CompareUtil, and so on?

Reply:
I change name to SortUtil and move it to sort package. (It has only 120 line now)